lua/tardis/parts/legacy/legacy_physbrake.lua

```diff
@@ -11,8 +11,8 @@ PART.Sound = "tardis/control_handbrake.wav"
 
 if SERVER then
 	function PART:Use(ply)
-		local result = self.exterior:TogglePhyslock() or false
-		ply:ChatPrint("Physics Lock ".. (result and "engaged" or "disengaged"))
+		self.exterior:TogglePhyslock()
+		ply:ChatPrint("Physics Lock ".. (self.exterior:GetData("physlock") and "engaged" or "disengaged"))
```

Comment:
This print should be dependent on the above TogglePhyslock returning true, maybe put a message in the else like "failed to toggle physlock"?

Reply:
TogglePhyslock() used to return current physlock state
Now it returns whether it was toggled successfully, as ToggleFloat() does
Basically, I didn't change anything - this is refactoring, not functionality change

Plus, this all will go away soon with centralized log/message system